lib/src/combined_wrappers/combined_list.dart

```diff
@@ -23,17 +23,17 @@ List/*<T>*/ combineLists/*<T>*/(List<List/*<T>*/> lists) {
   if (lists.length == 1) {
     return new UnmodifiableListView/*<T>*/(lists.first);
   }
-  return new _CombinedList/*<T>*/(lists);
+  return new CombinedList/*<T>*/._(lists);
 }
 
-class _CombinedList<T> extends ListBase<T> implements UnmodifiableListView<T> {
+class CombinedList<T> extends ListBase<T> implements UnmodifiableListView<T> {
```

Comment:
`CombinedListView`

Also, remove `combinedList()` in favor of a `CombinedListView` constructor.